### ISSUE TITLE
mx_MatrixToolbar_warning should have alt text

### ISSUE THREAD

**Issue by mgifford** [vector-im/riot-web#9537]
### Description

You can either use alternative text here:
`<img class="mx_MatrixToolbar_warning" src="img/warning.05cc423.svg" width="24" height="23">`

Or add a title in the SVG here:
https://riot.im/app/img/warning.05cc423.svg

Like described here:
https://www.deque.com/blog/creating-accessible-svgs/

### Steps to reproduce

I went here:
https://riot.im/app/#/room/!SOMErandomCODE:matrix.org

Ran the WebAim WAVE toolbar & looked at the source code.

<img width="615" alt="Screen Shot of Webaim Error" src="https://user-images.githubusercontent.com/116832/56475972-9a998800-645d-11e9-8f22-62ed36070346.png">

**Comment by lampholder** [comment]
I've put this as p2 because, if I understand correctly, it's the presence of the warning icon that will be undetectable to users using screen readers - they'll still be able to read the text of the warning. Is that right?

**Comment by mgifford** [comment]
I would add `alt="alert"` to the `<img>`.  That's probably the easiest.  You could argue that it is redundant to the existing text in which case you should use `alt=""` to tell screen readers that it is a decorative image. 

https://webaim.org/techniques/alttext/
